luigi/rpc.py

```diff
@@ -76,6 +76,26 @@ def fetch(self, full_url, body, timeout):
         return urlopen(full_url, body, timeout).read().decode('utf-8')
 
 
+class BobaPKIHTTPAdapter(requests.adapters.HTTPAdapter):
+    """HTTP adapter which disables hostname validation on HTTPS connections.
+
+    Copied from $ATT/rpc2/affirm/rpc2/transport/http.py
+    """
+
+    def init_poolmanager(self, *args, **kwargs):
+        super(BobaPKIHTTPAdapter, self).init_poolmanager(assert_hostname=False, *args, **kwargs)
```

Comment:
FYI this will break if this is called with `assert_hostname=<asd>` - do you want to pop it out of kwargs?

Reply:
this what our ATT code is doing so it's prolly fine but yeah I can pop it out too